examples/pty/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# PTY (pseudo-terminals) are available on most UNIX-like
+# systems but for simplicity we only check macOS and Linux.
+# Windows has an entirely different API for terminals which this
+# demo does not support yet.
+set(SUPPORTED_SYSTEMS Darwin Linux)
+if(NOT CMAKE_SYSTEM_NAME IN_LIST SUPPORTED_SYSTEMS)
+    message(WARNING "Skipping the PTY example which supports macOS and Linux only.")
```

Comment:
This says "skipping", but how is the skipping actually happening?

Reply:
Good catch, now fixed